ext/intl/php_intl.c

```diff
@@ -692,7 +624,7 @@ static const zend_function_entry intl_functions[] = {
 	PHP_FE( msgfmt_format, arginfo_msgfmt_format )
 	PHP_FE( msgfmt_format_message, arginfo_msgfmt_format_message )
 	PHP_FE( msgfmt_parse, arginfo_msgfmt_parse )
-	PHP_FE( msgfmt_parse_message, arginfo_numfmt_parse_message )
```

Comment:
it's just a temporary change until https://github.com/php/php-src/pull/4812 gets merged